java/arcs/core/host/ServiceLoaderHostRegistry.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+package arcs.core.host
+
+import arcs.core.sdk.Particle
+import java.util.ServiceLoader
```

Comment:
This definitely requires the JVM. Consider moving this file to `arcs.jvm.core.host`

Reply:
Done